controllers/profiles/object_creators.go

```diff
@@ -202,3 +208,54 @@ func ensureWorkflowSpecConfigMapMutator(workflow *operatorapi.KogitoServerlessWo
 		}
 	}
 }
+
+// workflowPropsConfigMapCreator creates a ConfigMap to hold the external application properties
+func workflowPropsConfigMapCreator(workflow *operatorapi.KogitoServerlessWorkflow) (client.Object, error) {
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      getWorkflowPropertiesConfigMapName(workflow),
+			Namespace: workflow.Namespace,
+			Labels:    labels(workflow),
+		},
+		// we could use utils.NewJavaProperties, but this way is faster
+		Data: map[string]string{
+			quarkusConfigFileName: "# Immutable Workflow Properties \n" +
```

Comment:
Why not load from a yaml file like other things created during the startup ? https://github.com/kiegroup/kogito-serverless-operator/tree/main/bundle/manifests

Reply:
@desmax74, Yeah, I thought about this over the weekend. The manifests are added by the bundle to the image, correct? So we have a path where we can look for?

Reply:
This configMap in this PR, can't be deployed by the manager since it will be one unique per workflow. 

What I'm looking for is to have an `application.properties` file in the bundle to load here, so we avoid hardcoding the default properties.

I'll take a look.

Reply:
@ricardozanini a trick could be load from the manager an application properties with the keys and placeholders as a values, every workflow define the values of the keys and after the replacement add it in the configmap or do you think is too complicated ?

Reply:
Loading a file every time is not the way to go, IMO. I've kept a var with the default properties. It's faster and more straightforward than either pinging the API or loading from a file.